packages/terra-notification-dialog/src/NotificationDialog.scss

```diff
@@ -0,0 +1,99 @@
+:local {
+  .notification-dialog {
+    background: var(--notification-dialog-background, #fff);
+    border-radius: var(--notification-dialog-border-radius, 0.3125rem);
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+
+    @media screen and (max-width: 768px) {
+      width: calc(100% - 2 * var(--notification-dialog-width, 10px));
+    }
+  }
+
+  .notification-dialog-body {
+    background-color: var(--notification-dialog-body-color, #fff);
+    display: inline-flex;
+    flex: 1 1 auto;
+    flex-direction: initial;
+    height: 100%;
+    overflow: auto;
+    padding: var(--notification-dialog-body-padding, 1.25rem);
+    width: 100%;
+  }
+
+  .notification-dialog-body-text {
+    background-color: var(--notification-dialog-body-color, #fff);
+    font-size: var(--notification-dialog-body-font-size, 1.25rem);
+  }
+
+  .header-body {
+    background-color: var(--notification-dialog-header-color, #f9f9f9);
+    border-color: var(--notification-dialog-header-border-color, #dedfe0);
```

Comment:
Group these into 1 border property. There is a bug with the sass-loader that mangles CSS custom properties in borders when it is split out into 3 individual properties.

e.g. `border: var(--notification-dialog-header-border, 0.0625rem solid #dedfe0);`